src/routes/transactions/entities/safe-app-info.entity.ts

```diff
@@ -6,11 +6,11 @@ export class SafeAppInfo {
   @ApiProperty()
   url: string;
   @ApiProperty()
-  logo_uri: string;
+  logoUri: string;
```

Comment:
Just fixing a typo here, since this is related to the mappings implemented in this PR.